db/migrate/20180705191153_add_upgrade_task.rb

```diff
@@ -0,0 +1,17 @@
+class AddUpgradeTask < ActiveRecord::Migration[5.1]
+  def change
+    create_table :upgrade_tasks do |t|
+      t.column :name, :string, :null => false
```

Comment:
I'm confused, what's name, task_name and subject and how they differ? Based on what I see in tests, subject seems as foreman, katello etc. If we enforce uniqueness, why we have subject too? Just trying to understand the purpose, I think it can stay as is.

Reply:
Name and task_name can be different because I may want to run some rake task when upgrading from version  A to version B and then again run the same task from version C to version D.   Otherwise we would have to create some new rake task just to make the 'task_name' value unique.